tests/lib/rules/ensure-storybook-title.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * @fileoverview Ensure title of a CSF matches the folder in which it is
+ * @author Benjamin Van Ryseghem
+ * @copyright 2022 finsit. All rights reserved.
+ * See LICENSE file in root directory for full license.
+ */
+
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+var rule = require("../../../lib/rules/ensure-storybook-title"),
+
+	RuleTester = require("eslint").RuleTester;
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+var ruleTester = new RuleTester();
+ruleTester.run("ensure-storybook-title", rule, {
+
+	valid: [
+		{
+			code: `export default {
+\tcomponent: ForecastCellInfoWidget,
```

Comment:
I would remove the `component` property in all but 1 example as this is unrelated and polluting